code/XFileExporter.cpp

```diff
@@ -514,12 +514,16 @@ void XFileExporter::WriteMesh(aiMesh* mesh)
 
 std::string XFileExporter::toXFileString(aiString &name)
 {
-	std::string str = std::string(name.C_Str());
-	std::replace(str.begin(), str.end(), '<', '_');
-	std::replace(str.begin(), str.end(), '>', '_');
-	std::replace(str.begin(), str.end(), '{', '_');
-	std::replace(str.begin(), str.end(), '}', '_');
-	std::replace(str.begin(), str.end(), '$', '_');
+	std::string pref = "NN_"; // node name prefix to prevent unexpected start of string
```

Comment:
Ok, no worries. I checked: our importer handles leading underscore correctly (or rather, it does not have a check to prevent it).

Please drop the prefix.